src/storage/mvcc/txn.rs

```diff
@@ -62,10 +62,15 @@ impl<'a> MvccTxn<'a> {
         }
     }
 
-    pub fn modifies(self) -> Vec<Modify> {
+    pub fn modifies(self, statistics: &mut Option<Statistics>) -> Vec<Modify> {
```

Comment:
Return `(Vec<Modify>, Statistics)` is more straightforward. It confuses me why it needs another `statistics` before seeing the implementation.